main.py

```diff
@@ -1,16 +1,13 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from logging.handlers import RotatingFileHandler
-from logging import Formatter, DEBUG
-from datetime import timedelta, datetime
```

Comment:
It looks like `datetime` is still used in `def internal_server_error(error):`

Reply:
I just re-added it and forced pushed an amend :)